README.md

```diff
@@ -141,10 +142,16 @@ The dashboard contains one widget per Step Function:
 
 ### ECS / Fargate
 
-ECS alarms are created for Fargate or EC2 clusters:
```

Comment:
Why did we remove "Fargate or EC2 clusters"?

Reply:
Sorry, it happened by by accident. I added back.